.github/workflows/gt4py-gitlab-ci.yml

```
@@ -2,7 +2,7 @@ name: CI
 on:
   push:
     branches:
-    - '**'
+    - master
```

I don't think that does what the plan needs to do. As far as I understands it needs also up-to-date branches.

---

Maybe push to master and pr to master works, but not sure. @DropD The goal is to run this action only on this repo, not on forks where it fails.

---

But it suspect that this solution doesn't work either because we need all the branches (associated to PRs) on gitlab to run the daint CI from there.

---

The branches in forks for PRs do not need to be mirrored on GitLab. Bors actually merges PR into the `trying` and `staging` branches to run the CI on GitLab, these ones do need to be mirrored though.

```suggestion
    - master
    - trying
    - staging
```